docs/recipes/ExpoRouter.md

```diff
@@ -1672,19 +1881,28 @@ We get that universal linking for free with `expo-router`!
 
 ## Code Cleanup
 
-Now that we have the boilerplate up and running again, let's clean some of the screen and navigation files that are no longer needed.
+Now that we have the boilerplate up and running again, let's clean some of the screen and navigation files that are no longer needed. We'll also remove the screen generator since there will not be one main directory for screens any longer
+
+:::note
+Currently the Ignite CLI doesn't have support for dynamic paths which would help generate screens in a filed-based routing system such as expo-router. We're currently looking into what it would take to support such functionality.
```

Comment:
```suggestion
Currently the Ignite CLI doesn't have support for dynamic paths, which would help generate screens in a file-based routing system such as expo-router. We're currently looking into what it would take to support such functionality.
```

In addition, "dynamic paths" is ambiguous, I might suggest something like:

> The screen generator is currently limited to generating files in a single destination directory, which means you'll still have to move screens manually when using expo-router. We're investigating ways to improve this experience.

Reply:
Thanks I like the sound of that better!